tests/autorag/data/parse/test_llamaparse.py

```diff
@@ -0,0 +1,68 @@
+from unittest.mock import patch
+
+import autorag
+from autorag.data.parse import llama_parse
+
+from tests.autorag.data.parse.test_parse_base import (
+	single_pdf_path_list,
+	multiple_pdf_data_list,
+	korean_text_glob,
+	eng_text_glob,
+)
+
+
+async def mock_llama_parse_pure(data_path: str, parse_instance):
+	return "Mocked OCR result", "mock_image_name"
```

Comment:
Is there any way you mock the llama parse function itself?
Not our function.
I mean you better mock `LlamaParse` library code

Reply:
There is no mock code in `LlamaParse` library 😂

Reply:
@bwook00 So that's why you should make one.
Who said you have to find mock code in `LlamaParse`? 
Like this I wrote a mock of `openai` library code.

```python
async def mock_gen_gt_response(*args, **kwargs) -> ParsedChatCompletion[Response]:
	return ParsedChatCompletion(
		id="test_id",
		choices=[
			ParsedChoice(
				finish_reason="stop",
				index=0,
				message=ParsedChatCompletionMessage(
					parsed=Response(answer="mock answer"),
					role="assistant",
				),
			)
		],
		created=int(time.time()),
		model="gpt-4o-mini-2024-07-18",
		object="chat.completion",
	)


@patch.object(
	openai.resources.beta.chat.completions.AsyncCompletions,
	"parse",
	mock_gen_gt_response,
)
def test_make_concise_gen_gt():

```